src/Interpreters/InterpreterCreateQuery.cpp

```diff
@@ -2082,18 +2082,18 @@ BlockIO InterpreterCreateQuery::doCreateOrReplaceTable(ASTCreateQuery & create,
         if (auto txn = current_context->getZooKeeperMetadataTransaction())
         {
             /// Avoid different table name on database replicas
-            UInt16 hashed_zk_path = sipHash64(txn->getTaskZooKeeperPath());
+            UInt64 hashed_zk_path = sipHash64(txn->getTaskZooKeeperPath());
             random_suffix = getHexUIntLowercase(hashed_zk_path);
         }
         else if (!current_context->getCurrentQueryId().empty())
         {
-            random_suffix = getRandomASCIIString(/*length=*/2);
-            UInt8 hashed_query_id = sipHash64(current_context->getCurrentQueryId());
+            random_suffix = getRandomASCIIString(/*length=*/8);
```

Comment:
It is generally ok. For me it is a bit strange to use mixed hex hash strings with lower case letters strings. But if this code was written before I don't think we need to fix this